ow_lander/src/ow_lander/common.py

```diff
@@ -52,5 +52,5 @@ def in_closed_range(val, lo, hi):
   """
   return val >= lo and val <= hi
 
-def create_most_recent_header(frame_id):
-  return Header(0, Time(0), frame_id)
+def create_header(frame_id, timestamp):
```

Comment:
How about making timestamp default to rospy.Time.now()? Then you won't have to put it on every line that calls create_header()

Reply:
`Time(0)` is a constant expression, but `rospy.Time.now()` is not. It will return whatever time it is now. If I did this the definition would have to look like this 
```suggestion
def create_header(frame_id, timestamp=None):
  if timestamp is None:
     return Header(0, rospy.Time.now(), frame_id)
```

I believe it is perfectly fine to define a function like this, but considering using the default value in this case hides the fact that when you call the function changes its return value in a way that is not very easy to communicate in the function name, I would much rather require a timestamp each time it is called.

Reply:
Thanks. I understand a little better what you're doing now. I still prefer default function parameters just to reduce code, but I don't prefer it strongly enough to hold up this PR.

Most of the time I would expect very little difference between using Time.now() and Time(0). Did you see any change in behavior from switching to Time.now()?

Reply:
Not really, just that `Time(0)` did not behave how the tf2 documentation described. Its meant to grab the "most recent" timestamp, but if that's what it indeed does then why would an error occur saying the frame does not exist? And why should increasing the timeout prevent that error from occuring? If it has to wait at all, then there is no way it is grabbing the most recent.

Since it was not behaving how the documentation said, I just felt it was safer to use something that does behave how the documentation describes e.g. `Time.now()`.